app/controllers/AddAdditionalMetadataController.scala

```diff
@@ -129,6 +129,33 @@ class AddAdditionalMetadataController @Inject() (
     consignmentService.getConsignmentFileMetadata(consignmentId, request.token.bearerAccessToken, Some(metadataType), Some(fileIds), additionalProperties)
   }
 
+  private def getMetadataValidation(consignmentId: UUID)(implicit request: Request[AnyContent]): Future[MetadataValidation] = {
```

Comment:
Would it make more sense to have this method in the `MetadataValidationUtils.scala`?

Reply:
I created this `MetadataValidationUtils.scala` to process data. I don't think it's a good idea to call services from the utility class.

Reply:
Fair enough